docs/MSPX.md

```diff
@@ -3,14 +3,16 @@
 ([back to main page](../README.md))
 
 mLRS provides the MspX technology, which is designed to improve the over-the-air communication for systems using the MSP protocol, specifically INAV. It includes the following features:
-- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV app or telemetry widgets to function.
+- MSP to CRSF message conversion which provides telemetry elements to the radio and therefore enables Lua scripts on the radio such as the INAV Telemetry Widget, FM2M Toolbox or the Horus Mapping Widget to function.
 - Robust framing and parsing which reduces packet losses to a minimum, and compression of some very large MSP messages to increase probability of successful transmission.
+- Providing of comprehensive mLRS link statistics and information to the Flight-Controller, to show them on the OSD and in blackbox recordings.
+- Possibility of RC-Control, serial data communication and Radio Telemetry through a single UART. 
 
 ***Notes***: 
-- MspX is not strictly compliant with the MSP protocol, and softwares which require strict compliance may not work properly. This includes e.g. MWP. In these cases, mLRS can be set into transparent mode by setting "Rx Ser Link Mode" = "transp." (the below features of MspX will then not be avaiable).
-- MspX is tested and verified for INAV 7.1. It has been successfully tested with INAV 8.0 development builds (it may also work with INAV 6.0 and newer, but this has not been tested).
+- For INAV Versions before 8 RC1 MspX is not strictly compliant with the MSP protocol, and softwares which require strict compliance may not work properly. This includes e.g. MWPT. In these cases, mLRS can be set into transparent mode by setting "Rx Ser Link Mode" = "transp." (the below features of MspX will then not be avaiable).
+- MspX is tested and verified for INAV 7.1 and newer (it may also work with older INAV releases, but this has not been tested).
 - INAV Configurator is assumed as ground control station, and INAV Lua app on the radio.
-- 'Link statistics and information such as RSSI, LQ, SNR, etc. are provided to the FC on INAV 8.0 and higher (not available on earlier versions). More info can be found [here](https://github.com/iNavFlight/inav/pull/10451).
+- Link statistics and information such as RSSI, LQ, SNR, etc. are provided to the FC on INAV 8 and higher (not available on earlier versions, RC Control over CRSF on a second UART are necessary, to recieve link statistics). More info can be found [here](https://github.com/iNavFlight/inav/pull/10451).
```

Comment:
I almost feel that can be deleted as well :) do we want to make any notes around earlier version of INAV?